gitbars/gitbars.py

```diff
@@ -96,10 +96,12 @@ def get_scores(items):
     return out
 
 
-def get_log(after, before, reverse=False, fill=False):
+def get_log(repo, after, before, reverse=False, fill=False):
     """Return the list of git log from the git log command."""
     # 2018-01-01 00:00:00|author@author.com
     args = ["git", "log", "--pretty=format:%ai|%ae", "--reverse"]
+    if repo:
+        args = [args[0]] + ["-C", f'{repo}'] + args[1:]
```

Comment:
This can be more concise. The default value of the repo arg can be "." and it can always be set in `args`, eliminating the `if` condition. So the repo is either `.` (current dir by default) or a custom path.